<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSetGameBoard</class>
 <widget class="QDialog" name="DialogSetGameBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="back.qrc">
    <normaloff>:/Recommended Source Files/Recommended Source Files/Images/mipmap-hdpi-v4/ic_launcher_round.png</normaloff>:/Recommended Source Files/Recommended Source Files/Images/mipmap-hdpi-v4/ic_launcher_round.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/Recommended Source Files/Recommended Source Files/Images/Back.jpg);</string>
  </property>
  <widget class="GameBoard" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>431</width>
     <height>441</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>381</width>
     <height>411</height>
    </size>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;BYRIL_Sea_Battle2&quot;;
background-color: rgb(0, 0, 255);
color: rgb(41, 134, 255);
background-image: url(:/Recommended Source Files/Recommended Source Files/Images/WhiteBackG.png);
selection-background-color: rgb(0, 170, 255);

gridline-color: rgb(48, 165, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::DragDrop</enum>
   </property>
   <property name="defaultDropAction">
    <enum>Qt::MoveAction</enum>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::DashDotLine</enum>
   </property>
   <property name="columnCount">
    <number>10</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>40</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>40</number>
   </attribute>
   <row>
    <property name="text">
     <string>A</string>
    </property>
    <property name="font">
     <font>
      <fontweight>Thin</fontweight>
     </font>
    </property>
   </row>
   <row>
    <property name="text">
     <string>B</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>C</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>D</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>E</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>F</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>G</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>H</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>I</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>J</string>
    </property>
   </row>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <item row="0" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="3" column="2">
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="DraggableButton" name="ship13">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>50</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/Recommended Source Files/Recommended Source Files/Images/ship1.PNG);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DraggableButton" name="ship12">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>100</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/Recommended Source Files/Recommended Source Files/Images/ship1.PNG);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DraggableButton" name="ship11">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>143</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/Recommended Source Files/Recommended Source Files/Images/ship1.PNG);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DraggableButton" name="ship21">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>185</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/Recommended Source Files/Recommended Source Files/Images/ship2.PNG);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DraggableButton" name="ship22">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>226</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/Recommended Source Files/Recommended Source Files/Images/ship2.PNG);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DraggableButton" name="ship23">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>270</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/Recommended Source Files/Recommended Source Files/Images/ship2.PNG);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DraggableButton" name="ship31">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>314</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/Recommended Source Files/Recommended Source Files/Images/ship3.PNG);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DraggableButton" name="ship32">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>358</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/Recommended Source Files/Recommended Source Files/Images/ship3.PNG);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="DraggableButton" name="ship41">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>400</y>
     <width>160</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/Recommended Source Files/Recommended Source Files/Images/ship4.PNG);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="randomSetButton">
   <property name="geometry">
    <rect>
     <x>553</x>
     <y>219</y>
     <width>81</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Recommended Source Files/Recommended Source Files/Images/WhiteBackG.png);
image: url(:/Recommended Source Files/Recommended Source Files/Images/flesh.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>500</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
background-image: url(:/Recommended Source Files/Recommended Source Files/Images/WhiteBackG.png);
background-color: rgb(85, 0, 255);
font: 25pt &quot;BYRIL_Sea_Battle2&quot;;
color: rgb(0, 0, 255);
}
QPushButton:hover {
 
	color: rgb(0, 85, 0);
	background-color: rgb(0, 85, 0);
}

</string>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>30</y>
     <width>31</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Recommended Source Files/Recommended Source Files/Images/WhiteBackG.png);
background-color: rgb(85, 0, 255);
font: 15pt &quot;BYRIL_Sea_Battle2&quot;;
color: rgb(0, 0, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DraggableButton</class>
   <extends>QPushButton</extends>
   <header>draggablebutton.h</header>
  </customwidget>
  <customwidget>
   <class>GameBoard</class>
   <extends>QTableWidget</extends>
   <header>gameboard.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="back.qrc"/>
 </resources>
 <connections/>
</ui>
